poolrpc/gen_protos.sh

```diff
@@ -6,19 +6,17 @@ set -e
 for file in ./*.proto
 do
 	protoc -I/usr/local/include -I. \
-	       -I$GOPATH/src/github.com/grpc-ecosystem/grpc-gateway/third_party/googleapis \
-	       --go_out=plugins=grpc,paths=source_relative:. \
-		${file}
+	  --go_out=plugins=grpc,paths=source_relative:. \
+		"${file}"
 
-done
-
-# Only generate the REST proxy and definitions for the client component.
-protoc -I/usr/local/include -I. \
-       -I$GOPATH/src/github.com/grpc-ecosystem/grpc-gateway/third_party/googleapis \
-       --grpc-gateway_out=logtostderr=true,paths=source_relative:. \
-       trader.proto
+  # Generate the REST reverse proxy.
+  protoc -I/usr/local/include -I. \
```

Comment:
Odd indentation here, is this intended to be a part of the for loop? If so should be indented to make it a bit easier to read imo.

Reply:
Was a mix of spaces and tabs, fixed.